internal/providers/terraform/azure/automation_job_schedule.go

```diff
@@ -1,60 +1,21 @@
 package azure
 
 import (
-	"fmt"
-
+	"github.com/infracost/infracost/internal/resources/azure"
 	"github.com/infracost/infracost/internal/schema"
-	"github.com/shopspring/decimal"
-	"github.com/tidwall/gjson"
 )
 
-func GetAzureRMAutomationJobScheduleRegistryItem() *schema.RegistryItem {
+func getAzureRMAutomationJobScheduleRegistryItem() *schema.RegistryItem {
```

Comment:
```suggestion
func getAutomationJobScheduleRegistryItem() *schema.RegistryItem {
```
Sorry, I missed this one.